src/test/java/com/solvd/carina/demo/MobileMWTest.java

```diff
@@ -0,0 +1,161 @@
+package com.solvd.carina.demo;
+
+import com.solvd.carina.demo.gui.mobilworld.android.*;
+import com.solvd.carina.demo.gui.mobilworld.android.components.Menu;
+import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.utils.R;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class MobileMWTest implements IAbstractTest {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    @Test
+    public void testLogin() {
+        LoginPage loginPage = new LoginPage(getDriver());
```

Comment:
I think that in tests you should operate on Base classes, so instead of LoginPage you should use LoginPageBase.
As there should be a single set of tests for all platforms. 
And now you have tests only for android with imports
import com.solvd.carina.demo.gui.mobilworld.android.*;
import com.solvd.carina.demo.gui.mobilworld.android.components.Menu;